modules/casper.js

```diff
@@ -214,6 +216,55 @@ var Casper = function Casper(options) {
         */
     });
 
+    // management ResourceTimeout with slimerJS
+    if (phantom.casperEngine === "slimerjs") {
+        this.checkTimeout = (function (data) {
+            var self = data;
+            return function(){
+                if ((!!this.page.settings.resourceTimeout) && (!isNaN(this.page.settings.resourceTimeout))) {
+                    var d = new Date().getTime();
```

Comment:
I'd like to see explicit variable names like "now" or "time".

This could be difficult to maintain, so the more explicit and easy to understand what is done is better :)